src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php

```diff
@@ -36,7 +36,10 @@ protected static function createClient(array $options = [], array $server = [])
         try {
             $client = $kernel->getContainer()->get('test.client');
         } catch (ServiceNotFoundException $e) {
-            throw new \LogicException('You cannot create the client used in functional tests if the BrowserKit component is not available. Try running "composer require symfony/browser-kit".');
+            if (class_exists(Client::class)) {
+                throw new \LogicException('You cannot create the client used in functional tests if the framework.test config is not set.');
```

Comment:
`You cannot create the client used in functional tests if the "framework.test" config is not set to true.`

Reply:
Updated :)